app/src/main/java/org/p2p/wallet/newsend/SendFeeRelayerManager.kt

```diff
@@ -44,6 +44,7 @@ class SendFeeRelayerManager(
     private lateinit var feeLimitInfo: FreeTransactionFeeLimit
     private lateinit var recipientAddress: SearchResult
     private lateinit var solToken: Token.Active
+    private var initializeCompleted = false
```

Comment:
I think we should fix the case in another way. 
This is a hack to fix but we need to find an issue